### PR TITLE
✨ Added proguard rules on for consumers of the library

### PR DESCRIPTION
**Pull request by andretortolano**
when minifying the library we need proguard rules to keep files in the data remote package so it can be used to generate the deserializers

### CODE REVIEW

**Comment by iujames** [comment]
linking to [sc-44253]

**Review by iujames — APPROVED** [review]
nice!

File: appcues/consumer-rules.pro [hunk 1]
```diff
@@ -0,0 +1,5 @@
+-keepclassmembers class com.appcues.data.remote.response.** {
+  <init>(...);
+  <fields>;
+  <methods>;
+}
```
Comment (iujames) [inline appcues/consumer-rules.pro:5]: maybe can simplify to 
```
-keep class com.appcues.data.remote.response.** { *; }
```
to keep everything in the JSON model package

Comment (andretortolano) [reply appcues/consumer-rules.pro:5]: I will try, thanks

**Pull request merged**